src/components/forms/ResortForm.js

```diff
@@ -0,0 +1,156 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Field, reduxForm, FieldArray } from 'redux-form';
+import { connect } from 'react-redux';
+import { getSpecificResort } from '../../store/resorts/selectors';
+import * as actions from '../../store/resorts/actionCreators';
+
+class NewResortForm extends Component {
+  componentDidMount() {
+    const { getResort, id } = this.props;
+    if (id) {
+      getResort(id);
+    }
+  }
+
+  renderField = ({
+    input, label, type, meta: { touched, error },
+  }) => (
+    <div>
+      <label>{label}</label>
+      <div>
+        <input {...input} type={type} placeholder={label} />
+        {touched && error && <span>{error}</span>}
+      </div>
+    </div>
+  );
+
+  renderPiste = ({ fields, meta: { error, submitFailed } }) => {
```

Comment:
What about extracting `renderField` and `renderPiste` to separate components?
Would it be problematic due to redux form?